biome.json

```diff
@@ -1,12 +1,45 @@
 {
```

Comment:
Please check the config and let me know if you'd add/edit any. @0xonramp @tokdaniel 

It should be pretty similar to what we had. The only changes as you can see from the altered files is mostly imports and colon for every end of line e.g. props in an object.